tests/testall.nim

```diff
@@ -8,4 +8,4 @@
 import testmacro, testsync, testsoon, testtime, testfut, testsignal,
        testaddress, testdatagram, teststream, testserver, testbugs, testnet,
        testasyncstream, testhttpserver, testshttpserver, testhttpclient
-import testutils
+import testutils, testratelimit
```

Comment:
`testutils` MUST BE last imported module for a reason, its why it is actually moved to different `import` statement, so please move `testratelimit` to previous `import` statement.

Reply:
what happens if it's not? this would be good to have in a comment, it's not obvious

Reply:
`testutils` runs tests/checks for pending Futures. So if there some pending futures happens in previous tests this last test will signal it. If you put other test after `testutils` this test could just bypass it.